<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subwin_check</class>
 <widget class="QWidget" name="subwin_check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="radioButton_checksum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>总加和</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_or">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>异或</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_crc32">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CRC32</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_crc16">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CRC16</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_crc8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CRC8</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm_checkwin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel_checkwin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
